analyzers/tests/SonarAnalyzer.UnitTest/TestFramework/Verifier.cs

```diff
@@ -289,5 +321,10 @@ public static void VerifyNoIssueReportedInTest(string path, SonarDiagnosticAnaly
 
         private static bool IsSupportForCSharp9InitNeeded(IEnumerable<ParseOptions> options) =>
             options != null && options.OfType<CSharpParseOptions>().Select(option => option.LanguageVersion).Contains(LanguageVersion.CSharp9);
+
+        private static IEnumerable<MetadataReference> AddTestReference(IEnumerable<MetadataReference> additionalReferences) =>
+            additionalReferences == null
+                ? NuGetMetadataReference.MSTestTestFrameworkV1
+                : NuGetMetadataReference.MSTestTestFrameworkV1.Concat(additionalReferences);
```

Comment:
Functionally the same, but less lines, without the `MSTEstTestFrameworkV1` repetition.
```suggestion
            NuGetMetadataReference.MSTestTestFrameworkV1.Concat(additionalReferences ?? Enumerable.Empty<>(MetadataReference));
```